src/shared/copy.h

```diff
@@ -30,6 +30,14 @@ typedef enum CopyFlags {
         COPY_GRACEFUL_WARN = 1 << 15, /* Skip copying file types that aren't supported by the target filesystem */
 } CopyFlags;
 
+typedef enum DenyType {
+        DENY_DONT = 0, /* we want INT_TO_PTR(DENY_DONT) to map to NULL */
```

Comment:
Not a big matter, but maybe, `DENY_UNSET` ?

Reply:
`DENY_DONT` seems more appropriate, since if we don't find a value, we don't want to deny the inode

Reply:
OK. That makes sense for me. Sorry for the noise.